modules/core/src/v2/coins/algo.ts

```diff
@@ -193,6 +232,57 @@ export class Algo extends BaseCoin {
     return [0, 1, 2];
   }
 
+  /**
+   * Builds a funds recovery transaction without BitGo
+   * @param params
+   * - userKey: [encrypted] Algo private key
+   * - backupKey: [encrypted] Algo private key, or public key if the private key is held by a KRS provider
+   * - walletPassphrase: necessary if one of the private keys is encrypted
+   * - bitgoKey : bitgo xpub of the wallet to recover funds from
+   * - krsProvider: necessary if backup key is held by KRS
+   * - recoveryDestination: target address to send recovered funds to
+   * @param callback
+   */
+  recover(params: RecoveryOptions, callback: NodeCallback<RecoveryTransaction>): Bluebird<RecoveryTransaction> {
+    return co(function*() {
+      // const userPrv = this.bitgo.decrypt({ input: params.userKey, passphrase: params.walletPassphrase });
+      // const backupPrv = this.bitgo.decrypt({ input: params.backupKey, passphrase: params.walletPassphrase });
+      // const bitgoPub = params.bitgoKey;
+
+      // const walletAddress = ''; // GET WALLET ADDRESS FROM  userPrv, backupPrv, and bitgoPub
+
+      // const walletInfoUrl = `algoexplorerhere.io/address/${walletAddress}`; // create the url for getting a wallet's info from a public block explorer
+
+      // // get wallet info from algorand public block explorer
+      // let walletInfo;
+      // try {
+      //   walletInfo = yield request.get(walletInfoUrl).result();
+      // } catch (e) {
+      //   throw new Error('Unable to reach the Stellar network via Horizon.');
+      // }
+
+      // const balance = 100; // get balance from walletInfo
+      // const txfee = 1; // set transaction fee here
+
+      // const algoTx = {}; // construct an algo tx that spends balance to params.recoveryDestination
```

Comment:
you can take this from buildTransaction on platform, this will provide a raw unsigned tx to feed to sign here